### PR TITLE
improve: ascanrulesAlpha: Add Top10 Mappings

### PR DESCRIPTION
**Pull request by kingthorin**
- Updated scan rules and unit tests.
- Renamed SpringActuatorScanRuleTest to SpringActuatorScanRuleUnitTest for consistency.
- Added note to CHANGELOG.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

File: addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/LdapScanRuleUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.zaproxy.addon.commonlib.CommonAlertTag;
+
+/** Unit test for {@link CorsScanRule}. */
```
Comment (thc202) [inline addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/LdapScanRuleUnitTest.java:30]: Needs update (same comment for following one).

File: addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/LdapScanRuleUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+class LdapScanRuleUnitTest extends ActiveScannerTest<LdapInjectionScanRule> {
```
Comment (thc202) [inline addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/LdapScanRuleUnitTest.java:31]: `LdapInjectionScanRuleUnitTest`

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**